Documentation/Installation/TutorialDdev.rst

```diff
@@ -45,7 +45,7 @@ of preconfigured projects.
 
 ..  code-block:: bash
 
-    ddev config --php-version 8.2
+    ddev config
```

Comment:
```suggestion
    ddev config --php-version 8.2
```

I would personally like to keep the PHP version here (even though ddevs default is 8.2 right now) because of two reasons:

- The version is aligned to the current TYPO3 version recommendation; the next LTS version may target PHP 8.3 and then we might miss adapting this "8.2" reference, if not there
- People may use older DDEV versions
- Future DDEV versions might use PHP 8.4 as a default, and when people then do not specify 8.2 here, they might have incompatibilities when using PHP 8.2